component-library/components/molecules/breadcrumbs/breadcrumbs.js

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+
+import classnames from "classnames";
+import breadcrumbs from "./breadcrumbs.scss";
+
+const propTypes = {
+    className: PropTypes.string,
+    navigation: PropTypes.shape.isRequired
```

Comment:
const propTypes = {
    className: PropTypes.string,
    navigation: PropTypes.shape({
        path: PropTypes.string.isRequired,
        name: PropTypes.string.isRequired
    }).isRequired
};
note: a shape type needs an object shape

Reply:
no wait I'm wrong because this is an array. Need an array with shapes

Reply:
```
navigation: PropTypes.arrayOf(
        PropTypes.shape({
            path: PropTypes.string.isRequired,
            name: PropTypes.string.isRequired
        })
    ).isRequired
```

okk like this